src/Compilers/CSharp/Portable/Compilation/AwaitExpressionInfo.cs

```diff
@@ -1,33 +1,30 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
-using Microsoft.CodeAnalysis.CSharp.Symbols;
+using System.Diagnostics;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.CSharp
 {
     /// <summary>
-    /// Structure containing all semantic information about an await expression.
+    /// Public structure containing all semantic information about an await expression.
```

Comment:
Perhaps remove `Public` since users of the public API will not see the `internal` type.